plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/history/HistoryPresenter.java

```diff
@@ -138,11 +138,24 @@ public void showDialog(Project project) {
     }
 
     /** Get the log of the commits. If successfully received, then display in revision grid, otherwise - show error in output panel. */
-    private void getCommitsLog() {
-        service.log(appContext.getDevMachine(), project.getLocation(), null, false).then(new Operation<LogResponse>() {
+    private void fetchHistoryPage(final boolean append) {
+        service.log(appContext.getDevMachine(),
+                    project.getLocation(),
+                    null,
```

Comment:
`null` is passed but `log` method's javadoc says nothing about it.
I think it would be nice to have documented this possibility.